bridges/centralized-ethereum/src/actors/dr_reporter.rs

```diff
@@ -68,12 +68,20 @@ impl DrReporter {
     }
 }
 
-/// Report the result of this data request id to ethereum
+/// Report the result of this data requests to ethereum
 pub struct DrReporterMsg {
+    /// Reports
+    pub reports: Vec<Report>,
+}
+
+/// Report the result of this data request id to ethereum
+pub struct Report {
     /// Data request id in ethereum
     pub dr_id: DrId,
-    /// Data Request Bytes
-    pub dr_bytes: Bytes,
+    /// Timestamp of the solving commit txs in Witnet. If zero is provided, EVM-timestamp will be used instead
+    pub timestamp: u64,
+    // Data Request Bytes
+    //pub dr_bytes: Bytes,
```

Comment:
if we are not using, should we remove it?

Reply:
Yes